webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricsSharedList.js

```diff
@@ -27,6 +23,10 @@ export class SakaiRubricsSharedList extends SakaiRubricsList {
     };
   }
 
+  connectedCallback() {
+    super.connectedCallback && super.connectedCallback();
```

Comment:
You don't need to test for the existence of connectedCallback on the super. It is definitely there.